docs/diginstroom/sip/1.0/profiles/newspaper.md

```diff
@@ -0,0 +1,346 @@
+---
+layout:       default
+title:        Newspaper
+parent:       Profiles
+nav_order:    2
+nav_exclude:  false
+---
+# Profile: Newspaper 
+
+The newspaper profile supports the ingest of digitised newspaper content.
+It is an extension of the generic SIP specification and it shows how to deal with multiple media files (in formats such as TIFF, ALTO XML and PDF) and the relationships between them and their metadata.
```

Comment:
profile instead of extension?

Reply:
Reformulated to 'It shows how to deal with multiple media files (in formats such as TIFF, ALTO XML and PDF) and the relationships between them and their metadata.'.